pkg/sql/temporary_schema_test.go

```diff
@@ -231,6 +231,14 @@ func TestTemporaryObjectCleaner(t *testing.T) {
 	)
 	defer tc.Stopper().Stop(context.Background())
 
+	{
+		// Create another empty database to ensure that cleanup still works in the
+		// presence of databases without temp objects. Regression test for #55086.
```

Comment:
would you mind adding this test to master as well? and perhaps a test in `pkg/sql/logictest/testdata/logic_test/temp_table` as well?

Reply:
I can add a test to master. Do the logic tests test the cleanup path, though? I thought we issued all the statements in a single session.

Reply:
ah yes you are right.